plugins/repository-hdfs/src/test/resources/rest-api-spec/test/secure_hdfs_repository/30_snapshot_readonly.yml

```diff
@@ -16,6 +16,7 @@
             security:
               principal: "elasticsearch@BUILD.ELASTIC.CO"
             readonly: true
+            conf.dfs.data.transfer.protection: "authentication"
```

Comment:
is this setting still needed with encryption being enabled? (see also all other occurrences of this)